keylime/tpm/tpm_main.py

```diff
@@ -1059,6 +1111,10 @@ def __tpm2_checkquote(self, pubaik, nonce, quoteFile, sigFile, pcrFile, hash_alg
                 "-q",
                 nonce,
             ]
+        else:
```

Comment:
This case should never happen. Calling sys.exit() is fine for the functions only used by the agent, but for the verifier i think it is not a good idea. Maybe instead we can only add the check for the old version and remove the check for the newer ones.

Reply:
do you mean something like that? 71b33a9
If it looks good to you, I will apply the same change to all the spots where I added that "internal error" thing.

Reply:
Yeah, seems the better option.

Reply:
great, I will apply the change everywhere.

Reply:
done